### PR TITLE
Solved ORM homework

### PR DESCRIPTION
**Pull request by GordanMasic**

### CODE REVIEW

**Comment by GordanMasic** [comment]
Napisati abstraktnu klasu "Model", koja sadrži dvije metode "String findByPk(Integer pk)" i "String findByAttribute(String a1, String a2)", koje na osnovu parametara kreiraju query string tipa "SELECT \* FROM {table_name} WHERE id={pk}".
Naziv tabele je potrebno proslijediti u klasama koje naslijeđuju klasu Model.

File: homeworkW13D5/src/homeworkW13D5/ORM/Model.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+public abstract class Model {
+
+	@SuppressWarnings("unused")
+	private String tableName;
+
+	public Model(String tableName) {
+		this.tableName = tableName;
+	}
+
+	public String findByPk(Integer id, String tableName) {
```
Comment (jasminb) [inline homeworkW13D5/src/homeworkW13D5/ORM/Model.java:11]: Why require table name. It is already known and stored in 'tableName' private member.

File: homeworkW13D5/src/homeworkW13D5/ORM/Model.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+		return String
+				.format("SELECT * FROM %s  WHERE id = %d ;", tableName, id);
+	}
+
+	public String findByAttribute(String tableName, String value) {
```
Comment (jasminb) [inline homeworkW13D5/src/homeworkW13D5/ORM/Model.java:16]: Same issue again. Method should take attribute name/value argument. Table name is already known to this class.

File: homeworkW13D5/src/homeworkW13D5/ORM/Person.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
+package homeworkW13D5.ORM;
+
+public class Person extends Model {
+
+	private String tableName;
+	private String value;
+	private Integer id;
+
+	public Person(String tableName, String value, Integer id) {
```
Comment (jasminb) [inline homeworkW13D5/src/homeworkW13D5/ORM/Person.java:9]: Constructor should not contain name of the table, it should be known in advanced and `hard-coded` in class constructor.

```
public Person(..) {
  super("person_tbl");
}
```

**Pull request merged**